tests/unit/sagemaker/serve/builder/test_model_builder.py

```diff
@@ -67,10 +67,18 @@
 class TestModelBuilder(unittest.TestCase):
     @patch("sagemaker.serve.builder.model_builder._ServeSettings")
     def test_validation_in_progress_mode_not_supported(self, mock_serveSettings):
```

Comment:
Can we add a test a where it is supported?

Reply:
Yes, I will add it. Good suggestion.